shopping-cart/shopping-cart-scala/shopping-cart/src/main/scala/com/example/shoppingcart/impl/ShoppingCartServiceImpl.scala

```diff
@@ -57,4 +59,16 @@ class ShoppingCartServiceImpl(persistentEntityRegistry: PersistentEntityRegistry
   private def convertShoppingCart(id: String, cart: ShoppingCartState) = {
     ShoppingCart(id, cart.items.map((ShoppingCartItem.apply _).tupled).toSeq, cart.checkedOut)
   }
+
+  /**
+   * Get a shopping cart report (view model).
+   *
+   * Example: curl http://localhost:9000/shoppingcart/123/report
+   */
```

Comment:
Prefer documenting the `Service` on the `shopping-cart-api` project only so we don't have to maintain docs in two places.

Reply:
I don't remember to have added this. I think it was IntelliJ. I will remove it.